app/layout.tsx

```diff
@@ -54,10 +55,7 @@ export default function RootLayout({
       <body className="default_background_color">
         <Providers>
           <Navbar />
-          <main className="mt-[48px]">
-            {children}
-          </main>
-          <Footer />
```

Comment:
You just deleted the footer from everywhere. We want to keep it, just not on the home page.